tests/users/test_api.py

```diff
@@ -295,3 +292,23 @@ async def test_remove_permission(spawn_client, role, status, snapshot):
 
     assert resp.status == status
     assert await resp.json() == snapshot()
+
+
+@pytest.mark.parametrize("first_user_exists, status", [(True, 409), (False, 201)])
+async def test_first_user_view(
+    spawn_client, mongo, first_user_exists, status, snapshot
+):
+    """
+    Checks response when first user exists and does not exist.
+    """
+    client = await spawn_client(authorize=True, administrator=True)
```

Comment:
Why? Given that this is the first user route I think the most realistic route would be to have a client with no perms at all.